week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code ", "your");
```

Comment:
Actually, you can use " or ' quote marks.  Most important is to be consistent with your team mates (or yourself), or the existing code of whatever project you're working on.

Reply:
ok